hahomematic/device.py

```diff
@@ -455,3 +456,13 @@ def create_devices(server) -> None:
         server.callback_system_event(
             HH_EVENT_DEVICES_CREATED, new_devices, new_entities
         )
+
+
+def _is_binary_sensor(parameter_data) -> bool:
+    """Check, if the sensor is a binary_sensor."""
+    if parameter_data[ATTR_HM_TYPE] == TYPE_BOOL:
+        return True
+    value_list = parameter_data.get("VALUE_LIST")
+    if value_list == ["CLOSED", "OPEN"] or value_list == ["CLOSED", "TILTED", "OPEN"]:
```

Comment:
`["CLOSED", "TILTED", "OPEN"]` ist kein `binary_sensor`. Es gibt User die tatsächlich zwischen `TILTED` und `OPEN` differenzieren. Die brauchen also wirklich einen `sensor`.

Reply:
Das ist mir bewusst. Problem ist, wenn es kein binary_sensor ist funktionieren die device_classes nicht (Icon, Übersetzung). Sensor werden werden leider nicht übersetzt.
Vielleicht `sensor` und `binary_sensor` anlegen?

Reply:
Ich würde in dem Fall eigentlich eher auf die device_class verzichten. Und hier muss ja auch nicht unbedingt übersetzt werden. Wenn es in HA nichts gibt was die Funktion des Gerätes 100%ig wiedergibt, dann würde ich da nix verbiegen. So wie das [aktuell](https://github.com/home-assistant/core/blob/49b04571729a38a604041f9fc96974ca927e7258/homeassistant/components/homematic/sensor.py#L48) ist finde ich das schon in Ordnung. Zumal gerad bei den anderen Sensoren mit der Value List die anderen Werte wirklich wichtig sind zur Differenzierung (z.B. beim CO2 Sensor).

Reply:
OK